src/content/react-hooks-slide-in-modal.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title: 'React Hooks Slide In Modal'
+author: 'Elliott Hindman'
+tags: ['advent2019', 'React', 'React Hooks', 'JavaScript']
+image: 'img/posts/react-hooks-slide-in-modal/banner.jpg'
+date: '2019-12-13T18:00:00.000Z'
+draft: false
+---
+
+# React Hooks Slide In Modal
+
+Continuing our studies of React Hooks, today I'll show you how to build a slide-in modal, which is useful for anything from a notification or popup to a settings screen for a user.
+
+If you missed my last article, please be sure to check out <a href="/react-hooks-color-picker">How to build a color picker using React Hooks</a>.
```

Comment:
@eliotc1986 

Anchor looks wrong.
"/blog/react-hooks-color-picker" is collect (Need to add "/blog")

Reply:
@yosi-q Good catch!
fixed here: 
https://github.com/yamagata-developers-society/blog/pull/29/commits/5c2646457a3e3ca4e7d07edbf1f8e25d15995c82#diff-9555d6f3ecbb565494081286b7acb7e4R15

Reply:
Confirmed!